app/scripts/migrations/131.test.ts

```diff
@@ -547,6 +655,14 @@ describe('migration #131', () => {
                               methods: [],
                               notifications: [],
                             },
+                            'wallet:eip155': {
+                              accounts: [
+                                'wallet:eip155:0xdeadbeef',
```

Comment:
I feel like this is a good illustration of why having CAIP-10 addresses in this array is silly. 🤷